packages/common/utils/url.js

```diff
@@ -0,0 +1,22 @@
+export const parseUrl = url => {
+  let prefix = '';
+
+  if (!/^https:\/\//.test(url)) {
+    prefix = 'https://';
+  } 
+    
+  try {
+    const urlObject = new URL(prefix + url);
+    
+    return urlObject;
+  } catch(error) {
+    console.warn(`URL ${url} is invalid`);
```

Comment:
You should avoid such warns because of spamming to logs when we have a lot of wrong items.

Reply:
Removed